lib/omniauth/openid_connect.rb

```diff
@@ -2,4 +2,5 @@
 
 require 'omniauth/openid_connect/errors'
 require 'omniauth/openid_connect/version'
-require 'omniauth/strategies/openid_connect'
+#require 'omniauth/strategies/openid_connect'
```

Comment:
Why is this commented out?

Reply:
Because `omniauth/strategies/openid_connect` is not a subdirectory. This is required from `lib/omniauth_openid_connect`.